x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/250_grok.yml

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
I'm adding these to runtime fields just to prove that everything is plugged in. I expect the scripts themselves to get simpler and simpler over time. But you can use grok for runtime fields after this PR!